code/style.css

```diff
@@ -1,4 +1,249 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/****html {
+  background: #7886d6;
+  color: rgb(42, 12, 92);
+}
+******/
+
+body {
+  background: #7886d6;
+  color: rgb(42, 12, 92);
+}
+
+/***************   HEADER    ************/
+
+header {
+    height: 60px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 15px; 
+    
+   }
+
+h2 {
+  padding-left: 20px;
+  padding-top: 10px;
+  font-family: Helvetica; 
+  font-size: 15px;
+  color: rgb(36, 233, 10);
+  width: 30%;
+}
+
+header img {
+    width: 50px;
+    padding-left: 10px;
+}
+
+nav {
+  padding: 20px;
+   
+}
+
+nav a {
+    color: #fff;
+    font-size: 10px;
+    text-decoration: none;
+    margin-left: 10px;
+  }
+
+/* NAV bar*/
+.sticky {
+  
+  position: fixed;
+  width: 92%;
+  background: navy;
+  display: flex;
+  justify-content: right;
+  border: 2px solid rgb(74, 250, 5);
+  box-shadow: 2px 5px 2px 2px rgba(34, 33, 33, 0.4);
+  }
+
+  
+/**************  MAIN NEWS     ****************/
+
+.big-news-container {
+    padding: 100px, 10px;
+    margin: 8px;
+    display: flex;
+    flex-direction: column;
+  }
+
+.big-news-container img {
+    width: 100%;
+    border: 2px solid black;
+    box-shadow: 2px 5px 2px 2px rgba(34, 33, 33, 0.4);
+
+}
+
+.main-news {
+  padding: 10px;
+  font-size: 18px;
+  text-align: justify;
+  }
+
+.big-news-child {
+  border: 2px solid black;
+  box-shadow: 2px 5px 2px 2px rgba(34, 33, 33, 0.4);
+  margin: 5px;
+  padding: 5px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  flex-grow: 1;
+}
+
+
+/****** OTHER NEWS     *****************/
+.flex-parent {
+  /*border: 2px solid blue;*/
+  height: 100%;
+  display : flex;
+  justify-content: center space-between 2px space-around 2px;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin: 5px;
+}
+
+
+.flex-child {
+  border: 1px solid rgb(21, 10, 51);
+  box-shadow: 2px 5px 2px 2px rgba(34, 33, 33, 0.4);
+  background:rgb(211, 189, 245);
+  display: flex;
+  margin: 5px;
+  width: 100%;
+  height: 100%;
+  flex-grow: 1;
+  
+  }
+
+
+
+
+.flex-child img {
+  padding: 2px;
+  box-shadow: 1px 2px 1px 1px rgba(63, 20, 59, 0.4);
+
+  width: 50%;
+  
+}
+
+.flex-child img:hover{
+  border: 3px solid rgb(251, 5, 190);
+  
+  padding: 10px;
+  margin: 5px;
+  box-shadow: 2px 5px 2px 2px rgba(80, 5, 72, 0.4);
+
+}
```

Comment:
I like the idea of this! The only problem is that the space is not allocated from the beginning, which creates a "jumping" of the text on hover. [Here](https://codepen.io/hippiekick/pen/zYoYPXr?editors=1100) is an example of how you can do it without the jumping